src/templates/sysconfig.html

```diff
@@ -661,5 +662,38 @@ <h3>{{ 'Adjust your privacy policy notice'|trans }}</h3>
     </div>
 </div>
 
+<!-- TAB 9 EXTERNAL AUTH -->
```

Comment:
please remove the trailing whitespaces
![2020-06-29-183117_925x558_scrot](https://user-images.githubusercontent.com/3043706/86031880-c7203b00-ba36-11ea-93bf-6346ef2192e4.png)